pages/en/exchange-faq.mdx

```diff
@@ -0,0 +1,132 @@
+by something such as systemd, so it will auto-restart.
+
+```sh
+MINA_STATUS=$($MINA client status --json)
+HIGHESTBLOCK="$(echo $MINA_STATUS | jq .highest_block_length_received)"
+HIGHESTUNVALIDATEDBLOCK="$(echo $MINA_STATUS | jq .highest_unvalidated_block_length_received)"
+# Calculate difference between validated and unvalidated blocks.
+# If block height is more than 4 block behind, somthing is likely wrong.
+DELTAVALIDATED="$(($HIGHESTUNVALIDATEDBLOCK-$HIGHESTBLOCK))"
+if [[ "$DELTAVALIDATED" -gt 4 ]]; then
+  $MINA client stop
+fi
+```
+
+## Why do users' funds appear to get lost when sending to some exchanges?
+
+Some exchanges require their users to include a _unique_ memo field when sending
+MINA deposits to the exchange, in order to associate the deposit with the user's
+account. If the user does not include this unique memo when sending their
+deposit, the receiving exchange may not be able to associate the deposit
+properly with the user's exchange account.
+
+_Note that these funds are **not** lost. They will be received at the exchange's
+address, but the exchange may not be able to automatically associate them with
+the user's exchange account without such a memo._
+
+Mina and its SDKs have full support for the `memo` field when sending a
+transaction.
+
+Exchanges & wallet creators can help avoid the above issue by exposing an
+optional `memo` field during a Mina send transaction.
+
+## Our archive node is missing block information after a restart. How can the data be recovered?
+
+Archive node operators often choose to run <DocLink copy="redundant archive nodes" url="/advanced/archive-redundancy" /> to store block data to one or more locations of their
+choice (e.g. PostgreSQL, GCP, local files, a logging service, etc) and to
+backfill any missed block data if needed.
+
+For convenience, O(1) Labs makes data from its archive node
+[available here](https://console.cloud.google.com/storage/browser/mina_network_block_data),
+to help others in the community backfill any missing information.
+
+This bucket contains blocks from various Mina networks—e.g. `mainnet` & the most
+recent DevNet `devnet2`. Filter the file names to find those for the network you
+desire. (Note that this bucket contains blocks for various other networks too,
+such as `qanet`, which we recommend _against_ using for your testing. `qanet` is
+used by O(1) Labs during their iterative development.)
+
+File names contain the network name, block height, and state hash of the block.
+Block height was added to the filename more recently, so blocks older than
+height 25,705 will include only the network name and state hash in the filename.
+
+Example file names:
+
+(Recent)
+
+`mainnet-30627-3NLfKanQ53X2MRKx5ZRvb9nVCEB9eJpcnssGCTpT3J1cojhB5M19.json`
+
+(Older)
+
+`mainnet-3NKUBmkc7UZ7ik5JyCM4WNzkN1HG5heMB5zNDUkf3Kgta1MFY6LY.json`
+
+You can download a specific block using curl:
+`curl https://mina_network_block_data.storage.googleapis.com/<filename.json>`.
```

Comment:
Nit: Could we make this a `code` section with the triple backticks?
```bash
curl https://mina_network_block_data.storage.googleapis.com/<filename.json>
```

Reply:
yeah, that's better